swagger.yaml

```diff
@@ -81,6 +81,8 @@ paths:
           description: Success
           schema:
             $ref: '#/definitions/message-response'
+        "208":
```

Comment:
Do we want to have specific code to state that registration was rejected?

Reply:
registration is succeded here :-)

Reply:
I understand we do not track the rejection the approve will stay in `false`. Let's make sure the device is not communicating at some point with exponential timeout or other means.